packages/react-ui-kit/src/Form/Input.tsx

```diff
@@ -95,7 +95,7 @@ export const Input: React.FC<InputProps<HTMLInputElement>> = React.forwardRef<
     />
   );
   return props.label ? (
-    <Label bold={false} markInvalid={props.markInvalid}>
+    <Label bold={false} markInvalid={props.markInvalid} style={{width: '100%'}}>
```

Comment:
Are you sure that this will not break other places where this component is already being used?

Reply:
Should be fine in most cases™️